@copilot+claude-sonnet-4.6 Could you fix the the workflow for [Test code / type_consistency (pull_request)](https://github.com/gdsfactory/gdsfactory/actions/runs/25846475887/job/75942751920?pr=4546)

For note, the current mypy error is
```
Run uv run mypy gdsfactory/
gdsfactory/component.py:1605: error: Unused "type: ignore" comment  [unused-ignore]
Found 1 error in 1 file (checked 446 source files)
Error: Process completed with exit code 1.
```

--- a/gdsfactory/component.py
+++ b/gdsfactory/component.py
@@ -1602,7 +1602,7 @@ class ComponentAllAngle(ComponentBase, kf.VKCell):
         polygon = points_to_polygon(points)
 
         res = self.shapes(_layer).insert(polygon)  # type: ignore[func-returns-value]
-        return res  # type: ignore[no-any-return]
+        return res
 
     def get_polygons(self, layer: LayerSpec) -> list[kf.kdb.DPolygon]:
         """Returns a list of polygons from the Component."""

